edgepi-leds.js

```diff
@@ -7,9 +7,17 @@ module.exports = function(RED) {
         node.ConfigStyle = config.ConfigStyle;
         node.LedPin = config.LedPin;
         node.Method = config.Method;
-        const led = new rpc.LEDService();
+        let led = null;
+        let transport = null;
 
-        console.log(node.Method, node.LedPin)
+        if(config.transport === "Network"){
+            transport = `tcp://${config.tcpAddress}:${config.tcpPort}`;
+            led = new rpc.LEDService(transport); 
+        } else {
+            transport = 'ipc:///tmp/edgepi.pipe';
+            led = new rpc.LEDService(transport);
+        }
```

Comment:
I think you should be able to take `led = new rpc.LEDService(transport);` outside of if-else

Reply:
63a226934951bf199ed7a8f4da11a89b006cfc62